modules/distributor/config.go

```diff
@@ -4,10 +4,10 @@ import (
 	"flag"
 	"time"
 
-	cortex_distributor "github.com/cortexproject/cortex/pkg/distributor"
 	"github.com/grafana/dskit/flagext"
 	"github.com/grafana/dskit/ring"
 	ring_client "github.com/grafana/dskit/ring/client"
+	tempo_distributor "github.com/grafana/tempo/pkg/distributor"
```

Comment:
no reason to give this an alias. just leave it `distributor`

Reply:
I just didn't like shadowing the package name.

Reply:
I didn't realize that you had copied these to `/pkg`. See comment here: https://github.com/grafana/tempo/pull/1251#discussion_r792660048